core/controller/root.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * core/controller/root.js
+ * Handles the dashboard and static pages
+ *
+ * @exports {Object} default - Koa router
+ */
+
+import Router from 'koa-router'
```

Comment:
then get rid of them everywhere else, i think. but don't do that now, do it in a separate PR that is only changing the semicolons.